matplotcheck/base.py

```diff
@@ -1102,7 +1102,13 @@ def assert_line(
             ):
                 flag_exist = True
                 line_x_vals = [coord[0] for coord in path_verts]
-                if min(line_x_vals) <= min_val and max(line_x_vals) >= max_val:
+                if (
```

Comment:
@nkorinek can you please add a comment here that explains how this fixes this issue. i understand the issue at a high level. so what if the data have a few extreme edge cases outlier points. and the regression line only fits to 3/4 of the data or 1/2 of the data. will this fail? i think i just need a bit more clarification 

did we discuss making the data limits of a line optional? that could be something else to consider OR creating a regression line and provided it to the method. i just want to talk this through a bit more so i understand and i want the code to have a comment for the same reason. thank you again!